### ISSUE TITLE
PyTorch 1.0 backend crashes on `_sparse_coo_tensor_unsafe`

### ISSUE THREAD

**Issue by zcwang0702** [dmlc/dgl#278]
## 🐛 Bug

Error in tutorial "Graph Convolutional Network"
```
---------------------------------------------------------------------------
~/ppython-virtual/env-3.6.3/lib/python3.6/site-packages/dgl/backend/pytorch/tensor.py in sparse_matrix(data, index, shape, force_format)
     25         raise TypeError('Pytorch backend only supports COO format. But got %s.' % fmt)
     26     # NOTE: use _sparse_coo_tensor_unsafe to avoid unnecessary boundary check
---> 27     spmat = th._sparse_coo_tensor_unsafe(index[1], data, shape)
     28     # No conversion is required.
     29     return spmat, None

AttributeError: module 'torch' has no attribute '_sparse_coo_tensor_unsafe'
```

## Environment

 - DGL Version : '0.1.2'
 - Backend Library & Version : PyTorch 1.0
 - OS : mac
 - How you installed DGL : pip
 - Python version: 3.6.3
 - no CUDA

**Comment by zcwang0702** [comment]
Closes #278

**Comment by BarclayII** [comment]
Should be fixed in release 0.1.3.
